pkg/controllers/kube-apiserver.go

```diff
@@ -125,6 +125,7 @@ func (s *KubeAPIServer) configure(cfg *config.MicroshiftConfig) error {
 			"proxy-client-key-file":            {cfg.DataDir + "/certs/kube-apiserver/secrets/aggregator-client/tls.key"},
 			"requestheader-client-ca-file":     {clientCABundlePath},
 			"service-account-signing-key-file": {cfg.DataDir + "/resources/kube-apiserver/secrets/service-account-key/service-account.key"},
+			"service-node-port-range":          {cfg.Cluster.ServiceNodePortRange},
```

Comment:
How is this different from the setting on line 171?

Reply:
I think line 171 (ServicesNodePortRange) was an internal struct that holds the value of service-node-port-range back in 4.1 code: https://github.com/openshift/origin/blob/release-4.1/pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go#L141

In latest openshift, it only exists in the vendor folder for setting the default port range value: https://github.com/openshift/origin/blob/master/vendor/k8s.io/kubernetes/openshift-kube-apiserver/configdefault/kubecontrolplane_default.go#L67, not sure if it is actually referenced in any other code.

[service-node-port-range](https://github.com/kubernetes/kubernetes/blob/master/cmd/kube-apiserver/app/options/options.go#L203) is the supported option in upstream kube-apiserver.

@deads2k Could you help confirm that ServicesNodePortRange is not used for specifying customized port range (I noticed from git history that you used to update code related to the service-node-port-range)?